src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Transfer/WhenDataLockEventsAreTransferred.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using CS.Common.SqlBulkCopyCat;
+using CS.Common.SqlBulkCopyCat.Model.Config.Builder;
+using SFA.DAS.Provider.Events.DataLock.IntegrationTests.Helpers;
+using SFA.DAS.Provider.Events.DataLock.IntegrationTests.TestContext;
+
+// this (commented out) test can be used to manually test the transer from transient db to Deds dbof DataLockEvents and related tables
+
+namespace SFA.DAS.Provider.Events.DataLock.IntegrationTests.Transfer
+{
+    class WhenDataLockEventsAreTransferred
+    {
+        //// comment out in GlobalSetup...
```

Comment:
Is this an instruction to anyone who wants to run the test? In that case, could you just pop it into a comment block above the test, ie. "Remove BeforeAllTests() in GlobalSetup before running".

Reply:
I've tidied up the file, including better instructions on how to run the test.